actions/email.go

```diff
@@ -26,6 +27,7 @@ type EmailAction struct {
 	Port       int
 	Sender     string
 	Recipients []string
+	LocalName  string
```

Comment:
is LocalName the right name for this variable? Not DomainName maybe (as derived from fqdn)?

Reply:
I agree that `LocalName` isn't a right name. It should be changed.
also `DomainName` can be misleading for email settings, isn't it?

maybe something like `client-host-name` ?